app/components/QuestionForm/QuestionTypeField.js

```diff
@@ -111,8 +111,10 @@ const QuestionTypeField = ({
           >
             <FormattedMessage {...messages[questionType.label]} />
           </Button>
+           
         ))}
       </ButtonGroup>
+      {error && <Warning>You need to select post type</Warning>}
```

Comment:
This text should be for a russian language too, please use translation json files